modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -14,6 +14,7 @@ The following procedure provides an example for scaling up the default Ingress C
 
 * You have the {oc-first} installed.
 * You have access to an {product-title} cluster as a user with the `cluster-admin` role.
+* On {vmw-first}, bare-metal, and Nutanix installer-provisioned infrastructure, scaling up Ingress Controller pods does not improve external traffic performance. To improve performance, you must manually configure a user-managed load balancer for your cluster and the load balancer must know what cluster nodes handle incoming traffic from the Ingress Controller.
```

Comment:
```suggestion
* On {vmw-first}, bare-metal, and Nutanix installer-provisioned infrastructure, scaling up Ingress Controller pods does not improve external traffic performance. To improve performance, you must manually configure a user-managed load balancer for your cluster. The load balancer must be configured for the cluster nodes that handle incoming traffic from the Ingress Controller.
```
not a merge blocker, but the sentence will hit a higher complexity on the FK scale than you want; and the LB can't "know"; something to consider for another day